lib/github/workflow.js

```diff
@@ -0,0 +1,21 @@
+const { Project } = require('../models');
+const transformWorkflows = require('../transforms/workflow');
+const reduceProjectKeys = require('../jira/util/reduce-project-keys');
+
+module.exports = async (context, jiraClient) => {
+  const { data: jiraPayload } = transformWorkflows(context.payload);
+
+  if (!jiraPayload) {
+    context.log({ noop: 'no_jira_payload_check_run' }, 'Halting further execution for check since jiraPayload is empty');
+    return;
+  }
+
+  await jiraClient.workflow.submit(jiraPayload);
+
+  const projects = [];
+  reduceProjectKeys(jiraPayload, projects);
```

Comment:
I don't think you created this util, but this is a bad pattern to create the array, give the reference to a function which then modifies it without your express knowledge.  The better functional way would be to have something like `const projects = reduceProjectKeys(jiraPayload);`